docs/reference/monitoring/production.asciidoc

```diff
@@ -128,10 +128,9 @@ a single {kib} instance to access both your production cluster and monitoring
 cluster.
 +
 --
-NOTE: If the monitoring cluster uses a SAML, Kerberos, PKI, or OpenID Connect
-realm, a dedicated {kib} instance is required. The security tokens that are used
-in these realms are cluster-specific and therefore cannot be used by a single
-{kib} instance to connect to both the production and monitoring clusters.
+NOTE: If you use SAML, Kerberos, PKI, or OpenID Connect realms on your
```

Comment:
__note:__ I left a note [in Kibana docs PR](https://github.com/elastic/kibana/pull/77938#discussion_r492497622), can we somehow mention [Token authentication provider](https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#token-authentication) here as well?

Reply:
Done, thanks!